packages/@ionic/utils-terminal/package.json

```
@@ -45,6 +45,8 @@
     "@types/jest": "^26.0.10",
     "@types/node": "~10.17.13",
     "@types/signal-exit": "^3.0.0",
+    "@types/slice-ansi": "^4.0.0"
```

```suggestion
    "@types/slice-ansi": "^4.0.0",
```
there is a missing comma